.github/workflows/ci.yaml

```diff
@@ -177,6 +181,15 @@ jobs:
                       bzlmodEnabled: true
                     - folder: e2e/rules_foo
                       bzlmodEnabled: true
+                include:
```

Comment:
oh. You can do an exclude & then include pattern with GHA 😎

Reply:
That was the best way I could find to only run a few core tests on v7, while also trying to reduce the number of times we repeat that version string.